stripper/ze_doom3_v2.cfg

```diff
@@ -600,7 +623,54 @@ modify:
 		"OnTrigger" "minigun_particle1Start12-1"
 	}
 }
-; ======= LOGIC AUTO FOR CUSTOM SIZED TRIGGERS =======
+; Prevent ZMs from walking into stage 3 boss arena (tp avoidance)
+modify:
+{
+	match:
+	{
+		"classname" "trigger_teleport"
+		"targetname" "telehell1"
+	}
+	delete:
+	{
+		"OnStartTouch" "logs1Kill21"
+	}
+}
```

Comment:
Is it not possible to just delay this further? The likely purpose here is saving entities, and there's only an additional gap of a few seconds that needs to be covered.

"stop zms from walking into tp free area after 1st lvl 3 boss dies" shouldn't be necessary anymore either if this fix works.

Reply:
Will change to a delay instead.
>stop zms from walking into tp free area after 1st lvl 3 boss dies

This change actually only stopped the cts from leaving the arena as opposed to affecting the zms. I decided to leave it though because if cts decided to leave the arena so they could kill zm items for example, they'd be outside the tp that was meant for them and they'd tp with zms.

Reply:
That's fine, but the comment should be updated then.